packages/luna/src/components/PlanCard/PlanCard.tsx

```diff
@@ -0,0 +1,206 @@
+          )}
+        >
+          {label === "most-common" ? intl("plan.label.most-common") : null}
+          {label === "most-valuable" ? intl("plan.label.most-valuable") : null}
+        </Typography>
+      ) : null}
+      <div className="tw-flex tw-items-center tw-gap-4">
+        <Typography
+          element="subtitle-2"
+          weight="bold"
+          className={cn(
+            "tw-mb-2",
+            primary ? "tw-text-natural-50" : "tw-text-natural-950"
+          )}
+        >
+          {title}
+        </Typography>
+        {discount ? (
+          <div className="tw-self-stretch">
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                "tw-text-brand-700 tw-py-[2px] tw-px-3 tw-rounded-[30px]",
+                primary ? "tw-bg-discount-primary" : "tw-bg-discount-default"
+              )}
+            >
+              {intl("plan.discount", { value: formatNumber(discount) })}
+            </Typography>
+          </div>
+        ) : null}
+      </div>
+      <Typography
+        element="tiny-text"
+        weight="semibold"
+        className={cn(
+          "tw-inline-block tw-mb-4",
+          primary ? "tw-text-natural-100" : "tw-text-natural-700"
+        )}
+      >
+        {description}
+      </Typography>
+      <Typography
+        element="h4"
+        weight="bold"
+        className={cn(
+          "tw-mb-2",
+          primary ? "tw-text-natural-50" : "tw-text-brand-700"
+        )}
+      >
+        {intl("plan.weekly-minutes", { value: formatNumber(weeklyMinutes) })}
+      </Typography>
+      <div className="tw-flex tw-gap-2 tw-items-end tw-flex-wrap">
+        <Typography
+          element="subtitle-1"
+          weight="bold"
+          className={cn(primary ? "tw-text-natural-50" : "tw-text-natural-950")}
+        >
+          {formatNumber(priceAfterDiscount, {
+            maximumFractionDigits: 2,
+          })}
+        </Typography>
+        <Typography
+          element="subtitle-1"
+          weight="bold"
+          className={cn(primary ? "tw-text-natural-50" : "tw-text-natural-950")}
+        >
+          {intl("labels.currency.egp")}
+        </Typography>
+        {discount ? (
+          <div className="tw-flex tw-gap-1">
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                primary ? "tw-text-natural-100" : "tw-text-natural-700"
+              )}
+            >
+              {intl("plan.instead-of")}
+            </Typography>
+            <Typography
+              element="tiny-text"
+              weight="regular"
+              className={cn(
+                "tw-block tw-relative",
+                primary ? "tw-text-destructive-400" : "tw-text-destructive-600"
+              )}
+            >
+              {intl("plan.price-before-discount", {
+                value: formatNumber(price),
```

Comment:
The price after discount should have precision of 2 decimal places (the same as the price).